soco/data_structures.py

```diff
@@ -723,6 +724,16 @@ class DidlItem(DidlObject):
         }
     )
 
+    @property
+    def full_album_art_uri(self):
```

Comment:
It seems slightly inconsistent that this name and `library.build_album_art_full_uri` each have the "full" in different places.

I have no opinion about which one is better, though :)